src/Sulu/Bundle/EventLogBundle/Infrastructure/Doctrine/Repository/EventRecordRepository.php

```diff
@@ -62,13 +68,51 @@ public function createForDomainEvent(DomainEvent $domainEvent): EventRecordInter
         return $eventRecord;
     }
 
-    public function add(EventRecordInterface $eventRecord): void
+    protected function getClassName(): string
```

Comment:
think i would not extract this into a method for now, the `$this->entityRepository` is already protected and can be accessed by a child class